multipaz-compose/src/commonMain/kotlin/org/multipaz/compose/Util.kt

```diff
@@ -1,5 +1,55 @@
 package org.multipaz.compose
 
 import androidx.compose.ui.graphics.ImageBitmap
+import org.multipaz.compose.camera.CameraFrame
 
 expect fun decodeImage(encodedData: ByteArray): ImageBitmap
+
+/**
+ * Extract an arbitrary geometry rectangular bitmap from the original bitmap.
+ *
+ * The method is used in [multipaz-mlkit] to crop and level the face image using MLKit face detection data
+ * as needed for further processing.
```

Comment:
That's not entirely true, we use this in the test app. I would just stick to saying that this can be used to crop and level a face.